tests/specs/watch.ts

```diff
@@ -300,5 +303,57 @@ export default testSuite(async ({ describe }, { tsx }: NodeApis) => {
 				expect(p.stderr).toBe('');
 			}, 10_000);
 		});
+
+		describe('include', ({ test }) => {
+			test('file path & glob', async () => {
+				const entryFile = 'index.js';
+				const fileA = 'file-a';
+				const fileB = 'directory/file-b';
+				await using fixture = await createFixture({
+					[entryFile]: `
+						import fs from 'fs/promises';
+						Promise.all([
+							fs.readFile('./${fileA}', 'utf8'),
+							fs.readFile('./${fileB}', 'utf8')
+						]).then(([a, b]) => {
+							console.log(a + ' ' + b);
+						}).catch(console.error);
+					`.trim(),
+					[fileA]: 'content-a',
+					[fileB]: 'content-b',
+				});
+
+				const tsxProcess = tsx(
+					[
+						'watch',
+						'--clear-screen=false',
+						`--include=${fileA}`,
+						`--include=${path.join(fixture.path, 'directory/*')}`,
+						entryFile,
+					],
+					fixture.path,
+				);
+
+				tsxProcess.stdout?.on('data', async (data: Buffer) => {
+					const chunkString = data.toString();
+					if (chunkString.includes('content-a content-b')) {
+						await fixture.writeFile(fileA, 'update-a');
+						await setTimeout(watchDebounce + 10);
+						await fixture.writeFile(fileB, 'update-b');
+					} else if (chunkString.includes('update-a update-b')) {
+						await fixture.writeFile(entryFile, 'console.log("TERMINATE")');
+					} else if (chunkString.includes('TERMINATE')) {
+						tsxProcess.kill();
+					}
+				});
+
+				const tsxProcessResolved = await tsxProcess;
+				const stdout = stripAnsi(tsxProcessResolved.stdout);
+				expect(stdout).toContain(`change in ./${fileA}`);
+				expect(stdout).toContain(`change in ./${fileB}`);
```

Comment:
Need to handle mixed path separators on windows. From test output:
```
Expected substring: "change in ./directory/file-b"
Received string:    "content-a content-b
12:18:14 AM [tsx] change in ./file-a Rerunning...
12:18:15 AM [tsx] change in ./directory\\file-b Restarting...
update-a update-b
12:18:16 AM [tsx] change in ./index.js Rerunning...
TERMINATE"
```

Would also be good to increase timeout between writing fileA and fileB since I'm not seeing `update-a content-b` in the output.